google/cloud/spanner_dbapi/cursor.py

```diff
@@ -85,6 +87,7 @@ def __init__(self, connection):
         self._row_count = _UNSET_COUNT
         self.lastrowid = None
         self.connection = connection
+        self.client_side_statement_executor = StatementExecutor(connection)
```

Comment:
I'd be curious to get a sense for what architecture you have in mind for `StatementExecutor` in the long term.  It seems like it's currently only trying to execute a few specific statement types.  Is the goal to eventually move more of the logic from this file into `StatementExecutor`, and have it be responsible for executing most types of statements?

Reply:
No, my plan is not to move any logic from this class to StatementExecutor (part of client_side_statement_executor module). StatementExecutor will have different functions to execute different client side statements only similar to https://github.com/googleapis/java-spanner/blob/main/google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionStatementExecutorImpl.java

Reply:
Hm, in that case I wonder if the class should be called `LocalStatementExecutor` or `ClientSideStatementExecutor` or something, to make that more clear?

Python does generally prefer shorter names, though.  So maybe that's overkill.  Anyway it sounds like you're thinking of dropping the class and just using the module.

Reply:
Yes dropped the class